### DISCUSSION TITLE
Pagination Header Info Missing

### DISCUSSION THREAD

**Discussion by vpanchal-code** [kimai/kimai#3979] [Questions & Answers] [answered]
I am testing API pagination and I notice that the headers are not included as per: https://www.kimai.org/documentation/api-pagination.html

I have Kimai version 1.30.11 installed and making a call to timesheets with size and page parameters a call on postman:
![image](https://user-images.githubusercontent.com/119960622/233165766-0cdb3f56-c68e-40ce-b21d-60b4aac83f24.png)

As you can see in the image, I get a response with the timesheet records, but not getting pagination headers. Is there a setting to enable pagination headers to be available?

**Comment by kevinpapst** [comment, marked as answer]
This is only available with the next major version 2.x

**Reply by vpanchal-code** [reply]
Thank you, I was not sure if I was doing something wrong.
